net.sourceforge.ddrescueview.yml

```diff
@@ -0,0 +1,36 @@
+app-id: net.sourceforge.ddrescueview
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+
+command: ddrescueview
+rename-icon: ddrescueview
+rename-desktop-file: ddrescueview.desktop
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+
+modules:
+  - 'shared-modules/gtk2/gtk2.json'
+  - name: ddrescueview_source
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/project/ddrescueview/Test%20builds/v0.4.5/ddrescueview-source-0.4.5.tar.xz
+        sha256: 57383c394e62612ce2a799438b00c6e3c465c31f9ba940e077f624e2e7028465
+      - type: file
+        path: net.sourceforge.ddrescueview.metainfo.xml
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        lazbuild source/ddrescueview.lpi
+      - install -Dm755 source/ddrescueview -t ${FLATPAK_DEST}/bin
+      - install -Dm644 resources/linux/applications/ddrescueview.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -Dm644 net.sourceforge.ddrescueview.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/
+      - install -Dm644 resources/linux/man/man1/ddrescueview.1 -t ${FLATPAK_DEST}/share/man/man1
+      - |
+        cd resources/linux/icons/hicolor/
+        for size in * ; do install -Dm644 "./${size}/apps/ddrescueview.png" -t "${FLATPAK_DEST}/share/icons/hicolor/${size}/apps/" ; done
```

Comment:
you could just copy them and use `rename-icon` in the top level of the manifest.

Reply:
this would also fix the icon name in the desktop file.

Reply:
oh its already done that way. but why not just 
```
install -d $FLATPAK_DEST/share
cp -R resources/linux/icons $FLATPAK_DEST/share/
```

Reply:
I'm not sure why I didn't do it this way.
Thanks for the review.